src/models/userSchema.ts

```diff
@@ -0,0 +1,27 @@
+import Joi from 'joi';
+
+export const userSchemaWithId = Joi.object({
+  id: Joi.alternatives().try(Joi.string().required(), Joi.number().integer().required()),
```

Comment:
```suggestion
  id: Joi.number().required(),
```

Reply:
this for handle the both the uuid and the id